docs/source/samples/24_tiny_yolo_v4_decoding_on_device.rst

```diff
@@ -0,0 +1,42 @@
+24 - RGB & TinyYoloV4 decoding on device
```

Comment:
rename to 22_2, and tiny-yolo-v3 to 22_1 as Brandon suggested.
https://luxonis.slack.com/archives/C01N61J6J90/p1613940283017000?thread_ts=1613938984.010800&cid=C01N61J6J90

Reply:
My bad, I thought he was talking about the openVino version - makes sense now.